lib/editor/components/ExceptionDateRange.js

```diff
@@ -169,13 +174,16 @@ export default class ExceptionDateRange extends Component<Props, State> {
             }}
             title={<i className='fa fa-times' />}
           >
-            <MenuItem eventKey='range'>Delete range</MenuItem>
-            <MenuItem eventKey='end-date'>Delete end date</MenuItem>
+            <MenuItem eventKey='range'>{this.messages('deleteRange')}</MenuItem>
+            <MenuItem eventKey='end-date'>{this.messages('deleteEndDate')}</MenuItem>
           </DropdownButton>
         </div>
         {validationState === 'error' && startMoment && endMoment
           ? <small style={{gridColumn: 'span 12'}}>
-            One or more dates between {startMoment.format('MM/DD/YYYY')} and {endMoment.format('MM/DD/YYYY')} appears in another schedule exception. Please choose another date.
+            {/* TODO: limit the number of exceptions displayed to 3 or so... */}
+            {validationErrors.map(err => {
+              return <div>{err.reason}</div>
+            })}
```

Comment:
Second time I'm seeing this. Should we extract to a new component?

Reply:
Good call. Interested to see your thoughts on it in https://github.com/ibi-group/datatools-ui/pull/965/commits/ca65fa9f2a3a2c5c1e145502aa55cac34d9ad5e7

Reply:
Not bad!